src/components/timers/Stopwatch.tsx

```diff
@@ -1,3 +1,39 @@
-const Stopwatch = () => null;
+import { useState, useEffect } from 'react';
+import HomeBtns from '../generic/HomeBtns';
+import TimerDisplay from '../generic/TimerDisplay';
 
+const Stopwatch = () => {
+    const [seconds, setSeconds] = useState(0);
+    const [isRunning, setIsRunning] = useState(false);
+
+    useEffect(() => {
+        let interval = null;
+
+        if (isRunning) {
+            interval = setInterval(() => {
```

Comment:
setInterval returns an interval that repeats every X ms. If you are going to clear the interval and recreate it each time, there's no point in using an interval but rather setTimeout, which runs just once.